application/server.py

```diff
@@ -86,13 +107,23 @@ def handle_join_room_form():
         room_name = request.form.get('room-name')
         room_password = request.form.get('room-password')
 
-        # TODO: Look for room in database
-
-        if room_password == "correct": # TODO: Verify password is correct
-            # TODO: create a new player in the database (you'll need the game's id)
-            return redirect('/waiting-room')
-        else:
-            return redirect('/enter-room')
+        games = models.Game.query.all()
+        
+        for game in games:
+            game = models.Game.query.filter_by(room_name=room_name, room_password=room_password).first()
```

Comment:
Not here. See previous comment. You shouldn't have to use a for loop to find the right game.